src/runtime/api-channel.ts

```diff
@@ -442,6 +456,7 @@ export abstract class PECOuterPort extends APIPort {
   @NoArgs Stop() {}
   DefineHandle(@RedundantInitializer store: StorageProviderBase, @ByLiteral(Type) type: Type, @Direct name: string) {}
   InstantiateParticle(@Initializer particle: recipeParticle.Particle, @Identifier @Direct id: string, @ByLiteral(ParticleSpec) spec: ParticleSpec, @ObjectMap(MappingType.Direct, MappingType.Mapped) stores: Map<string, StorageProviderBase>) {}
+  RebootParticle(@RebootInitializer particle: recipeParticle.Particle, @Identifier @Direct id: string) {}
```

Comment:
is `@RebootInitializer` a good name? `@RedundantInitializer` for example defines what this type of initializer does, not where it is used. You could imagine that some other api channel call may need to redefine the mapping, at which point using a `@RebootInitializer` in that different method will look weird - because it is not a reboot method. That person will probably figure out that we should rename it to describe what it does, so that we can use this different type of an initilizer across many calls. What do you think?

I would advise for something like `@OverridingInitializer`, `@RemappingInitializer` or `@RedeclaringInitializer`.

Reply:
That makes sense, I changed them to @OverridingInitializer. :)